src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -103,6 +103,8 @@ String processClassPath(String classPath) throws IOException {
                 .map(s -> {
                     if (s.indexOf("javafx-graphics") > 0) {
                         return javafxSDKLibsPath.resolve("javafx.graphics.jar").toString();
+                    } else if (s.indexOf("javafx-base") > 0) {
```

Comment:
That's okay for now, but maybe this method should just replace all the javafx jars? Also we have duplicated implementation on iOS.

Reply:
The base one should actually not be replaced. The reason I do it now is because I removed JFR dependency there. But that is better done via properties.
Actually, only the javafx-graphics one should contain specific Java code.